play/src/lib/PlayModels.tsx

```diff
@@ -35,25 +35,36 @@ export enum AnswerState {
 }
 
 /**
- * JoinGameState handles the substate changes for GameSessionState.JOIN_GAME
- * @enum JoingGameState
+ * OverallGameState handles the state changes between the Pregame or GameInProgress phases
+ * Pregame: user-controlled, joining the game
+ * GameInProgress: subscription-controlled, playing the game
+ * @enum OverallGameState
+ */
+export enum OverallGameState {
+  PREGAME = 'PREGAME',
+  GAME_IN_PROGRESS = 'GAME_IN_PROGRESS',
```

Comment:
You don't need to define string value always if you're not using it. Same for `HowToPlayMode`. If you don't need an absolute value for each case, you can omit defining it

Reply:
ugh, thanks. all string values removed here